src/libraries/System.IO.Pipelines/ref/System.IO.Pipelines.csproj

```diff
@@ -1,21 +1,23 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>$(NetCoreAppCurrent);netstandard2.0;net461</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent);netcoreapp3.1;netstandard2.0;net461</TargetFrameworks>
     <Nullable>enable</Nullable>
-    <ExcludeFromPackage Condition="'$(TargetFramework)' == 'net461'">true</ExcludeFromPackage>
     <!-- We only plan to use this ref in netcoreapp. For all other netstandard compatible frameworks
     we should use the lib asset instead. -->
-    <PackageTargetFramework>netcoreapp2.0</PackageTargetFramework>
-    <ExcludeCurrentNetCoreAppFromPackage>true</ExcludeCurrentNetCoreAppFromPackage>
+    <ExcludeFromPackage Condition="'$(TargetFramework)' != 'netcoreapp3.1'">true</ExcludeFromPackage>
```

Comment:
why we are building netcoreapp3.1 if we are not shipping it ?  why are adding net6.0 to the package ?

Reply:
We are excluding every ref except the netcoreapp3.1 one.

Reply:
and why are we adding the net6.0 config to the package ?